src/release/process.md

```diff
@@ -84,78 +111,56 @@ Send a PR to the master branch to:
 
 ## Release day (Thursday)
 
-Decide on a time to do the release, T.
-
-- **T-50m** - Run the following command in a shell with [AWS
-  credentials][awscli] in the [simpleinfra] repository:
-
-  ```
-  ./release-scripts/promote-release.py release prod stable
-  ```
+Decide on a time to do the release. Let the Social Media coordinator (currently Mara)
+know of the time, so that she can be ready to post the release on the project's social
+media channels.
 
-  That'll, in the background, schedule the `promote-release` binary to run on
-  the production secrets (not the dev secrets). That'll sign everything, upload
-  it, update the html index pages, and invalidate the CDN. Note that this takes
-  about 30 minutes right now. This will also push a signed tag to rust-lang/rust.
+As of September 2024 a release takes between 75 and 90 minutes to complete, so
+start the release process earlier enough to hit the time you planned.
 
-- **T-2m** - Merge blog post.
+To start the release, Run this command in the [rust-lang/release-team]
+repository[^auth]:
 
-- **T** - Tweet and post everything!
-
-  - Twitter [@rustlang](https://twitter.com/rustlang)
-  - [Users forum](https://users.rust-lang.org/)
+```
+./scripts/start-release.py publish-rust-prod-stable
+```
 
-- **T+5m** - Release and tag Cargo. From a rust-lang/rust checkout (script will
-  checkout the stable branch automatically), run the following script from
-  [simpleinfra].
+The command will start a background job to invoke [`promote-release`] targeting
+the production environment, and it will show the instructions to follow its
+logs.
 
-  ```sh
-  ../simpleinfra/release-scripts/tag-cargo.sh
```

Comment:
Should this tag-cargo.sh script be deleted from simpleinfra, or is there any reason to keep it?